guide/principles/principles.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: guide
+title: Principles
+description: The key principles to follow when designing Bitcoin products.
+nav_order: 2
+has_children: false
+permalink: /guide/principles/
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+The key principles of designing for Bitcoin
+
+-->
+
+# Bitcoin design principles
+
+As a new technology, Bitcoin offers the opportunity of a decentralized open financial system, where participants share the role of securing the network. This is important to give everyone equal and direct access without needing intermediaries. To make this a reality, we encourage everyone working on products to deliberately support the core principles of designing for Bitcoin. These are principles we in the Bitcoin Design Community have identified and stand behind. Some of these come from the technology itself, and others from the behaviour and ethos of the community. Although every use case and product is different, applications should strive to follow these principles. Diverging from them should only be done with very good reason.
+
+
+- [Decentralization](#decentralization)
+- [Inclusion](#inclusion)
+- [Transparency](#transparency)
+- [Self-custody](#self-custody)
+- [Security](#security)
+- [Privacy](#privacy)
+- [Interoperability](#interoperability)
+
+---
+
+## Decentralization
+
+> Encourage users to run their own Bitcoin node and connect directly to the network
+
+Unlike traditional banking systems, the Bitcoin economy does not require new users to seek permission from anyone. Bitcoin has no central point of control. No one person or entity is in charge. Connecting to any [node]({{ '/guide/glossary/#node/' | relative_url }}) on the network gives you the same rights and responsibilities, which also ensures that there is no single point of failure.
```

Comment:
If append `/` at the end of the link it doesn't redirects to the header. this fixes it.
```suggestion
Unlike traditional banking systems, the Bitcoin economy does not require new users to seek permission from anyone. Bitcoin has no central point of control. No one person or entity is in charge. Connecting to any [node]({{ '/guide/glossary/#node' | relative_url }}) on the network gives you the same rights and responsibilities, which also ensures that there is no single point of failure.
```